src/app/footer/footer.component.html

```diff
@@ -85,14 +84,43 @@
           </li>
         </ul>
       </div>
+      <div class="col-sm-4">
+        <ul class="footer-list m-0">
+          <li class="m-0 footer-heading">
+            Legal
+          </li>
+          <li>
+            <a href="https://github.com/ga4gh/dockstore/blob/1.5.4/LICENSE" target="_blank" rel="noopener noreferrer" class="footer-link"
```

Comment:
It was like this, but this should point to the latest license.

Not sure if it's worth the effort, but ideally this would be dynamic, pointing to the license of the current web service version, whatever it is.

The footer already displays the web service version (I don't know how it gets it), so you could presumably use that to dynamically generate this link.

OTOH, our license probably doesn't change that much, so it may not be worth the effort.

Also, it could be a separate ticket if others think it a good idea rather than part of this PR -- I just happened to notice because this line changed.

Reply:
`ts-node git.version.ts` writes the `src/app/footer/versions.ts`

Reply:
Hm, the links actually don't work I'm realizing now. 404s

Reply:
I see -- it's the version of the web service the UI was built against.

We could add the web service version to the config.json (or maybe it's already exposed some other way), and detect if there is a difference. Didn't this come up once in the past, like around in the 1.5 timeframe? I have a vague memory of something being out of sync.

I'll log that separately as a wish list item.

Reply:
Oops, the webservice version is gotten from the metadata endpoint, not the versions.ts

Reply:
I'll close the other ticket then. :)

Reply:
I know this was a long thread :), but can you please either fix this or log a ticket. We shouldn't be linking to a license on an older branch.